sys/linux/dev_infiniband_rdma_cm.txt

```diff
@@ -132,7 +132,7 @@ rdma_ucm_accept {
 
 rdma_ucm_reject {
 	id			rdma_cm_id
-	private_data_len	int8[0:RDMA_MAX_PRIVATE_DATA]
```

Comment:
`RDMA_MAX_PRIVATE_DATA` is 256 so we're actually asking for all possible `int8` values here. I'm not sure what the intent was here, so not sure I've got the correct fix. Should `private_data_len` be the bytesize of `private_data`?